OperatingSystem-Services/Service-SOC/BasicDocumentation/DockerInstall-Wazuh.md

```diff
@@ -0,0 +1,48 @@
+# How to install Docker For Wazuh Purposes
+
+## DISCLAIMER: Do not follow docker docs, follow provided steps
```

Comment:
Probably want to use a comment block rather than a header!

```
> [!IMPORTANT]
>DISCLAMER:  Do not follow docker docs, follow provided steps!

Reply:
Yeah do that.